operations/fetch_test.go

```diff
@@ -46,15 +42,25 @@ func TestClone(t *testing.T) {
 		}},
 	}
 
+	opts := cloneOptions{
```

Comment:
Can opts be defined in the for loop? This maintains the pattern of consistency / not requiring tests to run in a particular order.

Reply:
I tried that but moving the opts definition into the for loop causes the test to fail :(